### PR TITLE
Fix #2481: Display failures on top in html runner

### LINKED ISSUE #2481: Add easy way to skip to failures in HTML runner
Currently, one sees that there are failures, but one has to scroll down manually.

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

**Comment by gzm0** [comment]
Review by @sjrd

File: test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala [hunk 1]
```diff
@@ -379,6 +396,7 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
       var className: String = js.native
       val style: Style = js.native
       var onclick: js.Function0[Boolean] = js.native
+      def insertAdjacentElement(location: String, element: Element): Unit = js.native
```
Comment (gzm0) [inline test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala]: Is it OK to use this?

Comment (sjrd) [inline test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala]: According to the [browser compat table of MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML), I hope so.

File: test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala [hunk 2]
```diff
@@ -242,14 +250,19 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
       def log(msg: String, clss: String): dom.Element =
         body.newElement(clss = s"log $clss", text = msg, tpe = "pre")
 
+      def setNextSibling(that: TestBox): Unit = {
+        this.box.insertAdjacentElement("afterEnd", that.box)
```
Comment (sjrd) [inline test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala]: [On MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML) the doc says to use `"afterend"` all-lowercase. I suggest we do not tempt fate with alternative casing.

**Comment by sjrd** [comment]
That's all.

**Comment by gzm0** [comment]
Updated. Tested locally with more than one failing test, with and without test selection.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**